.gitignore

```diff
@@ -1,9 +1,9 @@
 *.log
 *.swp
-node_modules/
-public/
-coverage/
+node_modules
+public
+coverage
 **/.DS_Store
-.publish/
+.publish
```

Comment:
even though I prefer without them because it looks cleaner, you've got a great point! I'll add them back :)